internal/core/src/segcore/segment_c.cpp

```diff
@@ -207,12 +207,17 @@ Insert(CSegmentInterface c_segment,
        const uint64_t data_info_len) {
     try {
         auto segment = static_cast<milvus::segcore::SegmentGrowing*>(c_segment);
-        auto insert_data = std::make_unique<milvus::InsertData>();
-        auto suc = insert_data->ParseFromArray(data_info, data_info_len);
+        auto insert_record_proto =
+            std::make_unique<milvus::InsertRecordProto>();
+        auto suc =
```

Comment:
might introduce too much copy here.
but so far is good

Reply:
This is only a naming change from `using InsertData = proto::segcore::InsertRecord;` to `using InsertRecordProto = proto::segcore::InsertRecord;`. the name `InsertData` is overloaded for several purposes and ambiguous.